.github/workflows/build-publish-pypi-test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       uses: FragileTech/bump-version@main
       with:
         current_version: "${{ env.current_version }}"
-        files: setup.cfg
+        files: bumpversion.cfg
```

Comment:
read the docs. I am pretty sure you should not be changing this.

Reply:
Ok, I did some reading and you're right. It makes sense since we're trying to update the version field stored in setup.cfg.